tests/unit/BloomFilterSyncTest.cpp

```diff
@@ -0,0 +1,127 @@
+/* This code is part of the GenSync project developed at Boston University.  Please see the README for use and references. */
+//
+// Created by Anish Sinha on 12/4/23.
+//
+
+#include "BloomFilterSyncTest.h"
+#include <GenSync/Syncs/GenSync.h>
+#include <GenSync/Syncs/BloomFilterSync.h>
+#include "../TestAuxiliary.h"
+CPPUNIT_TEST_SUITE_REGISTRATION(BloomFilterSyncTest);
+
+BloomFilterSyncTest::BloomFilterSyncTest() = default;
+
+BloomFilterSyncTest::~BloomFilterSyncTest() = default;
+
+void BloomFilterSyncTest::setUp() {
+	const int SEED = 93;
+	srand(SEED);
```

Comment:
If you're using ntl's Random... functions, you should use it's SetSeed here instead, no?